src/rest.ts

```diff
@@ -148,6 +148,38 @@ ${queryParams
   }
 }
 
+function createScopedRequestHandler(
+  this: unknown,
```

Comment:
From the previous experience `unknown` produces an error when using TypeScript in a strict mode. Can we prefer `any` in these cases?

Reply:
Oh, this typing is not necessary, can be skipped here I think.